### DISCUSSION TITLE
office 365 setup

### DISCUSSION THREAD

**Discussion by jimmyeao** [moralmunky/Home-Assistant-Mail-And-Packages#750] [Q&A]
I have set up an azure app / secret as per the wiki, but there are no instructions on how to use it? IMAP auth is not working for Office 365.

**Comment by firstof9** [comment]
It's still a WIP

**Reply by callumeveratt** [reply]
@firstof9 Is there any idea when it will come? Assuming it requires code changes rather than just working out config steps due to the nature of app registrations?

**Reply by firstof9** [reply]
Yes it will be quite a bit of code change.
No ETA.

**Comment by firstof9** [comment]
Please see https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/discussions/726#discussioncomment-4627077
